plugins/azure-sites/src/api/AzureSitesApiBackendClient.ts

```diff
@@ -39,25 +39,33 @@ export class AzureSitesApiBackendClient implements AzureSitesApi {
       request.subscription
     }/${request.resourceGroup}/${request.name}/stop`;
     const { token: accessToken } = await this.identityApi.getCredentials();
+    const entity = request.entity;
     await fetch(url, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
         ...(accessToken && { Authorization: `Bearer ${accessToken}` }),
       },
+      body: JSON.stringify({
+        entity,
+      }),
```

Comment:
You'll need to send just an `entityRef` here instead, and then have the backend fetch the entity from the catalog instead. Otherwise the user can just forge an entity that allows them to take actions towards a site that they don't own.